### PR TITLE
[BEAM-2887] Changes for making wordcount example work with fnapi

### PR DESCRIPTION
**Pull request by aaltay**
R: @robertwb 
cc: @kennknowles

### CODE REVIEW

**Comment by kennknowles** [comment]
Look at #3941?

**Comment by aaltay** [comment]
Thank you @kennknowles. I missed #3941. I will revert the bundle_processor changes here.

**Comment by robertwb** [comment]
Given that this file is the primary example (from a documentation perspective), I'd like to avoid cluttering it up with (temporary) fn-api-related workarounds. Can we introduce a separate pipeline instead? (Better yet, if the goal is to aid development, can we make a test?)

**Comment by aaltay** [comment]
Thank you @robertwb.

What do you think about this version? Separated wordcount on fnapi pipeline and added a post commit integration test.

File: sdks/python/apache_beam/examples/wordcount_it_test.py [hunk 1]
```diff
@@ -64,6 +65,18 @@ def test_wordcount_it(self):
...
+  def test_wordcount_fnapi_it(self):
+    test_pipeline = TestPipeline(is_integration_test=True)
+
+    pipeline_verifiers = [PipelineStateMatcher()]
+    extra_opts = {'experiment': 'beam_fn_api',
+                  'on_success_matcher': all_of(*pipeline_verifiers)}
+
+    # Get pipeline options from command argument: --test-pipeline-options,
+    # and start pipeline job by calling pipeline main function.
+    wordcount_fnapi.run(test_pipeline.get_full_options_as_args(**extra_opts))
```
Comment (robertwb) [inline sdks/python/apache_beam/examples/wordcount_it_test.py]: Any reason not to just write

test_pipeline.get_full_options_as_args(
    experiment='beam_fn_api',
    on_success_matcher=PipelineStateMatcher())

Comment (aaltay) [reply sdks/python/apache_beam/examples/wordcount_it_test.py]: Thank you, fixed.

**Comment by aaltay** [comment]
I could not figure out how to branch wordcound_fnapi.py from the original with history. Do you know how?

I will try a bit to get the new test working, then merge this.

**Pull request closed without merging**